docs/tutorials/typescript-deploy.md

```diff
@@ -0,0 +1,206 @@
+# Deploy you smart contract to the Koinos blockchain
+
+Originally published by Roamin on [Hashnode](https://hashnode.com/@Roamin)
+
+Today we're going to learn how to deploy a smart contract to the Koinos blockchain. We won't be writing any code, so if you don't have a compiled Koinos smart contract ready to deploy, I invite you to check my previous blog where we learned how to "[Create your first Koinos smart contract in minutes](/first-smart-contract-on-koinos.md)".
+
+Like most other blockchains, Koinos has a testnet available (the current one is called Harbinger), so we're going to use it to create a test account, top it up with some tKoin, which is the native test token of the blockchain, and deploy our smart contract.
+
+## Koinos CLI installation
+
+The Koinos team provides a CLI that will help you manage your keys/wallets and also will allow you to interact with smart contracts. The official repository is hosted at [github.com/koinos/koinos-cli](https://github.com/koinos/koinos-cli).
+
+First, let's download the CLI that's compatible with our Operating System (OS), to do that:
+
+simply head over to the release page of the Github repo [github.com/koinos/koinos-cli/releases](https://github.com/koinos/koinos-cli/releases)
+
+download the `.zip` file that's compatible with your system.
```

Comment:
Capitalize "D"

Reply:
Replace `.zip` with "archive file," don't we offer `.tar.gz` files for non windows machines?